ui/lib/core/addon/mixins/replication-actions.js

```diff
@@ -88,16 +88,10 @@ export default Mixin.create({
     if (action === 'disable') {
       yield this.onDisable();
     }
-    if (mode === 'secondary' && replicationMode === 'dr') {
```

Comment:
conditionally I had earlier when just solving for the DR secondary transition issues.  I now pull this conditional into the onEnable method defined in the replication-mode controller.

Reply:
got it. so the transition issue only happened when enabling replication?

Reply:
Correct, all the transition issues I solved for were for enabling.